src/EditorFeatures/Core/InheritanceMargin/InheritanceMarginServiceHelpers.cs

```diff
@@ -354,9 +355,8 @@ internal static class InheritanceMarginServiceHelper
             targetSymbol = symbolInSource ?? targetSymbol;
 
             // Right now the targets are not shown in a classified way.
-            var definition = await targetSymbol.ToNonClassifiedDefinitionItemAsync(
+            var definition = await targetSymbol.ToSlimDefinitionItemAsync(
                 solution,
-                includeHiddenLocations: false,
                 cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
instead of an extension, you can just either have this method locally.

Reply:
or, could we jsut pass a flag to ToNonClassifiedDefinitionItemAsync saying if we need the display parts or not?

Reply:
I am assuming your suggestion is to merge the DefinitionItem extension method in this PR to 'ToNonClassifiedDefinitionItemAsync'
Yes that's possibile

Reply:
I will make it private first, because I feel this DefinitionItem only applied to Inheritance Margin